code/game/machinery/rechargestation.dm

```diff
@@ -99,6 +138,7 @@
 
 /obj/machinery/recharge_station/open_machine(drop = TRUE, density_to_set = FALSE)
 	. = ..()
+	sendmats = FALSE //Leaving off for the next user
```

Comment:
this seems like it'll be confusing for new players to figure out, why shouldn't it send mats automatically when new players enter?

Reply:
I'd like it to be an intentional decision for a borg to take station mats, since now they are sharing with the whole crew. 

Others players can also right-click the charger to turn on restocking, and there's a tip in the examine text. Between the two, I'm hoping it won't be a huge roadblock for new players.

FWIW, borgs can also refill their mats by clicking on an iron stack in the world with their held stack.

Reply:
I personally just don't like the idea that you have to manually do this every time you enter a recharger, this is not something we should be expecting both new and old borg players to know.

Reply:
agreed, was this addressed?